src/Microsoft.AspNetCore.Razor.Runtime/Runtime/TagHelpers/TagHelperExecutionContext.cs

```diff
@@ -159,7 +159,7 @@ public void Add(ITagHelper tagHelper)
         }
 
         /// <summary>
-        /// Tracks the minimized HTML attribute in <see cref="AllAttributes"/> and <see cref="HtmlAttributes"/>.
+        /// Tracks the minimized HTML attribute in AllAttributes and HtmlAttributes.
```

Comment:
`/// Tracks the minimized HTML attribute.`

Good catch here, we no longer expose HtmlAttributes or AllAttributes so might as well not expose them indirectly in doc comments. Same elsewhere.